devops/README.md

```diff
@@ -0,0 +1,19 @@
+# DevOps tools
```

Comment:
I think we can. It might help people who want to set up testnets. Anything that we wanted private (keys, some of the infrastructure elements) should move to a private repo - as soon as we can open another private repo.